lib/common/versionUtils.js

```diff
@@ -21,7 +21,7 @@ const _ = require('lodash'),
  * @returns {object} the resultant regular expresion using the provided data
  */
 function getVersionRegexp({ key, version }) {
-  return new RegExp(`${key}['|"]?:\\s?[\\\]?['|"]?${version}`);
+  return new RegExp(`${key}['|"]?\\s*:\\s*[\\\]?['|"]?${version}`);
```

Comment:
Does it works with tabs, not spaces?

Reply:
@dhwaneetbhatt Yes, I also have added test case for having a tab. Basically all whitespace chars are under `\s`.

Ref:
![image](https://github.com/postmanlabs/openapi-to-postman/assets/16756296/4aa19155-8e8d-4f46-a1d3-7dae4b9bc3d7)